components/pkg-export-helm/src/main.rs

```diff
@@ -84,6 +84,7 @@ fn cli<'a, 'b>() -> clap::App<'a, 'b> {
         .arg(
             Arg::with_name("VERSION")
                 .value_name("VERSION")
+                .short("v")
```

Comment:
We use `V` elsewhere for version, and `v` for `verbose`; we should remain consistent here, too.

Reply:
Sure, fixed.